### DISCUSSION TITLE
how to use WellKnownFolderName

### DISCUSSION THREAD

**Discussion by Irracionald73** [ecederstrand/exchangelib#1087] [Q&A] [answered]
Can you help me with explanation how to convert code  message.Save(new FolderId(WellKnownFolderName.Drafts, new Mailbox(strUserName)) to use with exchangelib ?

**Comment by ecederstrand** [comment, marked as answer]
Is this a new or existing message? If it exists, you can do `some_message.move(account.drafts)`. If it's a new folder, just set the `folder` attribute before calling `.save()`: `Message(folder=account.drafts, ...).save()`

**Reply by Irracionald73** [reply]
New Mailbox is not same as account, seem like i need to get some folder id from common credentials for specific user

**Reply by ecederstrand** [reply]
I'm not sure you can move items between accounts, but you can give it a try. Just create an `Account` object for the other account, and do `some_message.move(some_other_account.drafts)`.
